src/core/xfa/template.js

```diff
@@ -3425,6 +3426,7 @@ class Image extends StringObject {
     }
 
     // TODO: Firefox doesn't support natively tiff (and tif) format.
+    // TODO: Same about application/octet-stream
```

Comment:
The comment is useless: the type of the image will be guessed with the first bytes of the stream.